molgenis/molgenis-emx2
# chore(catalogue): fix network staging

What are the main changes you did:
- fix network staging profile yaml > removed CohortsStaging tag
- add demo data for network staging area
- add profile to data models (DataModels.java)
- edit TestLoader from 'general' to 'profile'

how to test:
- go to preview https://preview-emx2-pr-4425.dev.molgenis.org/apps/central/#/ and log in as admin
- make new schema with template DATA_CATALOGUE_NETWORK_STAGING
- see that template loads and ontologies load (is a dependent setting)

todo:
